types/chromecast-api/lib/device.d.ts

```diff
@@ -0,0 +1,207 @@
+import { EventEmitter } from "events";
+
+export interface DeviceOptions {
```

Comment:
this unfortunately, have to be rewritte,

so you should have, AFAIK: 
```ts
namespace Device {
    interface DeviceOptions {
    ...
    interface MediaResource {
    ...
}

declare class Device ...

export = Device;
````
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-class-d-ts.html